### DISCUSSION TITLE
no gdbserver with gdb?

### DISCUSSION THREAD

**Discussion by mellery451** [Homebrew/discussions#659] [Linux]
it seems like the gdb formula does not currently install gdbserver on linux (haven't checked mac). It looks like configure is called with `--enable-targets=all`, so I'm not sure why gdbserver isn't being built/installed. Does anyone happen to know why it's not installed?

**Comment by gromgit** [comment]
I've submitted a PR to fix this: https://github.com/Homebrew/homebrew-core/pull/70284

**Reply by mellery451** [reply]
very cool - thanks for submitting that.
